convert_numeric

```diff
@@ -0,0 +1,192 @@
+#!/bin/bash
+#
+#                         License
+#
+# Copyright (C) 2024  David Valin dvalin@redhat.com
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+# Convert various time and memory units.
+
+usage()
+{
+	echo "usage $1 <unit>"
+	echo "-h --help --usage: This help message"
+	echo "--cnv_unit <u>: Unit to convert to."
+	echo "--value <value>: Value to be converted."
+	echo "--unit <u>: Unit of value to be converted."
+	if [[ $1 == *"convert_numeric"* ]]; then
+		echo "If just --unit is passed, program returns the unit value.  EG.  --unit K will return 1000B"
+	else
+		echo "If just --unit is passed, program returns the unit value.  EG.  --unit us will return 1000ns"
+	fi
+	echo "If --value and --unit, program returns the value in the base unit."
+	echo "If --value, --unit and --cnvt unit is passed, then the program will convert <value><unit> to"
+	echo "   the unit designated in --cnv_unit"
+	echo ""
+	echo "Example usage:"
+	echo "$ ./convert_numeric --unit M --value 1024 --cnv_unit G"
+	echo "1G"
+	echo "$ ./convert_numeric --unit M --value 1024 --cnv_unit K"
+	echo "1024000K"
+	echo "$ ./convert_numeric --unit Mi --value 1024 --cnv_unit Ki"
+	echo "1048576Ki"
+	echo "./convert_numeric --unit Mi --value 1024 --cnv_unit Gi"
+	echo "1Gi"
+	echo "./convert_numeric --unit Mi"
+	echo "1000000B"
+	echo "$ ./convert_time --unit s --value 10 --cnv_unit ms"
+	echo "10000ms"
+	echo "$ ./convert_time --unit s --value 60 --cnv_unit m"
+	echo "1m"
+	echo "$ ./convert_time --unit s"
+	echo "1000000000ns"
```

Comment:
Wouldn't this be confusing to see 2 different script names within the same usage message?

I think adding a `--time` arg would make more sense then changing modes based off the script basename.